.github/workflows/vale-linter.yml

```diff
@@ -0,0 +1,19 @@
+name: Vale Linting
+
+on: pull_request
+
+permissions:
+  contents: read
+
+jobs:
+  prose:
+    if: github.repository == 'keptn-sandbox/new-keptn-docs-engine'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+
+      - name: Vale
+        uses: errata-ai/vale-action@c4213d4de3d5f718b8497bd86161531c78992084
```

Comment:
same as `checkout`: Any reason for using a sha1 hash instead of a release tag?

Reply:
@imsingh Like there is no specific reason to use sha1 instead of `errata-ai/vale-action@c4213d4de3d5f718b8497bd86161531c78992084` both will work fine. I followed some unofficial docs on vale linter and they have used sha1 so I thought that is the standard way to use it.